client/src/components/ProfileCard/ProfileCard.tsx

```diff
@@ -11,22 +7,22 @@ export default function ProfileCard(): JSX.Element {
   const classes = useStyles();
 
   return (
-    <Card className={classes.root} elevation={4}>
-      <CardContent>
+    <Paper className={classes.paper} elevation={4}>
+      <Box className={classes.box}>
```

Comment:
You can assign flexbox props to the `<Box>` component.

Eg: `<Box display="flex" alignItems="center">`